tests/test_gamma.py

```diff
@@ -26,10 +26,8 @@ def test_sd():
 )
 def test_from_quantiles(x0_x1):
     x0, x1 = x0_x1
-    if x0 == x1:
-        return
-    if x0 > x1:
-        x1, x0 = x0, x1
+    assume(abs(x1 - x0) > 1e-6)
+    assume(x0 < x1)
 
     q = (0.15, 0.95)
```

Comment:
Wouldn't (0.05, 0.95) be more natural? Or is there any reason for having 0.15 ?

Reply:
Yes typo, I guess. I didn't touch that in this PR, but go ahead and change it, if you want.

Reply:
Maybe it is not too important, then.